src/DynamoCoreWpf/Extensions/ViewModelCommandExecutive.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Dynamo.ViewModels;
+
+namespace Dynamo.Wpf.Extensions
+{
+    /// <summary>
+    /// The ViewModelCommandExecutive provides access to DynamoViewModel and WorkspaceViewModel commands
+    /// </summary>
+    public class ViewModelCommandExecutive
+    {
+        private DynamoViewModel dynamoViewModel;
+
+        /// <summary>
+        /// Create a Command Executive for a DynamoViewModel
+        /// </summary>
+        /// <param name="viewModel"></param>
+        internal ViewModelCommandExecutive(DynamoViewModel viewModel)
+        {
+            dynamoViewModel = viewModel;
+        }
+
+        /// <summary>
+        /// Fit the current workspace view to the current selection
+        /// </summary>
+        public void FitViewCommand()
+        {
+            dynamoViewModel.FitViewCommand.Execute(null);
+        }
+
+        /// <summary>
+        /// Search for an element by its ID and focus the view on it
+        /// </summary>
+        /// <param name="objectID"></param>
+        public void FindByIdCommand(string objectID)
+        {
+            dynamoViewModel.CurrentSpaceViewModel.FindByIdCommand.Execute(objectID);
```

Comment:
@mjkkirschner @QilongTang this is what we want instead of FitView. It focuses the view on the element, without changing the zoom. So we won't have the issue of causing users to lose a sense of where they are. 

Only problem is, it appears to be broken... 

It eventually calls this: [https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml.cs#L816-L848](https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml.cs#L816-L848)

As far as I can tell, this command isn't called anywhere in Dynamo. Which may explain its "only kind of working" state. It's kind of hard to explain what I mean by "only kind of working", but here it goes: currently, this command will focus the view on the element. Great. However, once the user goes to perform any other change to the view (pan, zoom in, zoom out, findByIdCommand, etc.), the view "forgets" it has been refocused on the element, and jumps back to its state before [CenterViewOnElement](https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml.cs#L816-L848) was called. The behavior is even stranger when you use the findByIdCommand several times in a row. I'm not exactly sure what this problem is. Continuing to investigate. Considering making a fix to this problem as part of this PR.

Reply:
Fixed: https://github.com/DynamoDS/Dynamo/pull/9944/commits/26077520ff81b84f9ad02051a82d9f290554a2ea

It was updating the view but not the model.